integration-tests/utils/integration-helpers.js

```diff
@@ -80,6 +80,18 @@ const addUserToBilling = _.flow(withSignedInPage, withUserToken)(async ({ page,
   }, email, billingProject)
 
   console.info(`added user to: ${billingProject}`)
+
+  const userList = await page.evaluate((billingProject) => {
+    return window.Ajax().Billing.project(billingProject).listUsers()
+  }, billingProject)
+
+  function getval(email) {
+
+    let obj = userList.filter(item => item.email === email)
+    return obj[0].role
```

Comment:
We shouldn't have to hard index into the filter result. This is very risky as it could cause the entire web application to crash. It's better to map the filtered result and display that.

Reply:
I didn't end up needing to map this. Should I be accounting for needing to map?

Reply:
No, the find covers it 👍